test/form/Form.spec.jsx

```diff
@@ -85,8 +85,7 @@ function render (props) {
           <TestField name="address.street" className="street" />
           <div>{`pristine ${pristine}`}</div>
           <div>{`submitting ${submitting}`}</div>
-        </div>
-      }
+        </div>}
```

Comment:
Wow. Good thing we don't have to discuss formatting anymore, 'cause I would be fighting against this to my death. 😆

Reply:
Oh, this looks like a formatting "error" to me (it actually was auto-formatted via `react/jsx-curly-newline`).
I dislike it so much that I added braces now (force-pushed; see [the diff](https://github.com/ePages-de/react-components/compare/543b6418f998f0a948b64efeb4843fcd53e6dd42..43b194527b493d371c28221a22f2337ec065c705)).